index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+	<title>Alvin's site 🔥</title>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+	<link rel="stylesheet" href="assets/css/main.css" />
+	<noscript>
+		<link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+</head>
+
+<body class="is-preload">
+
+	<!-- Wrapper-->
+	<div id="wrapper">
+
+		<!-- Nav -->
+		<nav id="nav">
+			<a href="#" class="icon solid fa-home"><span>Home</span></a>
+			<a href="#work" class="icon solid  fa fa-folder-open"><span>About me </span></a>
+			<a href="https://www.youtube.com/watch?v=BjDebmqFRuc" class="fa fa-user-secret"><span>secret link</span></a>
+			<a href="https://github.com/Alvinmash" class="icon brands fa-github"><span>Git-Hub</span></a>

```

Comment:
The standard on the web is for links within your site to stay in the same tab but links to other sites throw a new tab.